### PR TITLE
add testing capability

### PR DESCRIPTION
**Pull request by bemeyert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,6 +99,30 @@ Push specific docker images:
...
+
+### `test.sh`
+
+If a script called `test.sh` resides inside `<imagename>/tests` it is executed.
+`test.sh` is necessary when some preparations have to be made to run the Bats
+tests. Thus first all preparations are run and then the Bats tests.
+If there are any Bats tests they won't be executed by the test itself. Exceution
+must be done inside `test.sh`.
+
+One example is an image which delivers a nerworked service like a HTTP-server.
```
Comment (kr1sp1n) [inline README.md]: Typo: `nerworked`

File: docker-test/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+RUN ln -s /opt/bats/bin/bats /usr/sbin/bats \
+    && apk add --quiet --no-progress --no-cache python3 curl jq \
+    && /tmp/install_docker-compose.sh \
+    && rm /tmp/install_docker-compose.sh
+#
+# DEFAULTS
+#
+ENV IMAGE_PREFIX=""
+ENV IMAGES=""
+ENV CURRENT_BRANCH=""
```
Comment (kr1sp1n) [inline docker-test/Dockerfile]: Is `CURRENT_BRANCH` and `ONLY_BRANCH` needed in `docker-test` image?

File: README.md (hunk 1 above)
Comment (bemeyert) [reply README.md]: fixed

File: docker-test/Dockerfile (hunk 2 above)
Comment (bemeyert) [reply docker-test/Dockerfile]: removed

**Comment by bemeyert** [comment]
@kr1sp1n All comments fixed. Please approve

File: .travis.yml [hunk 3]
```diff
@@ -5,7 +5,16 @@ services:
 
 install: true
 
+env:
+  - DOCKER_COMPOSE_VERSION=1.24.0
+
 before_script:
```
Comment (kr1sp1n) [inline .travis.yml]: Please pick cherries from: https://github.com/sevenval/dockerfiles/blob/b041b69b5c1694f062f179e8e5f65dec9903b88f/.travis.yml

Comment (bemeyert) [reply .travis.yml]: I had to copy the stuff.

**Pull request merged**